### PR TITLE
Include tags in monitor definition

### PR DESCRIPTION
**Pull request by ngetahun-dd**
Currently tags are ignored in the monitor. Added the inclusion of tags.

### CODE REVIEW

File: tmpl/monitor.tmpl [hunk 1]
```diff
@@ -1,6 +1,7 @@
 resource "datadog_monitor" "{{ .Id }}" {
   name               = "{{ .Name }}"
   type               = "{{ .Type }}"
+  {{ if .Tags }}tags               = {{ .Tags | commaSeparateList }}{{- end }}
```
Comment (amnk) [inline tmpl/monitor.tmpl]: Wdyt about doing the same with built in functions? 
```
{{ range .Tags }}
{{ .... }}
{{ end }}
```

Comment (ngetahun-dd) [reply tmpl/monitor.tmpl]: Good point, I could have

```
tags = [
{{ range .Tags }}
{{ .... }},
{{ end }}
]
```

let me test it out

**Comment by ngetahun-dd** [comment]
@amnk Can you take another look? I updated it with your notes

**Comment by amnk** [comment]
@ngetahun-dd Thanks. This looks much better now!

**Pull request merged**